tpl/tplimpl/embedded/templates/shortcodes/youtube_simple.html

```diff
@@ -8,8 +8,12 @@
 {{ end }}
 {{ $secondClass := "s_youtube_simple" }}
 {{ $divID := printf "%s_%d" $secondClass (add $.Ordinal 1) }}
+{{ $tb := printf "//i.ytimg.com/vi/%s/" $id }}
+{{ $mx := printf "%smaxresdefault.jpg" $tb }}
+{{ $hq := printf "%shqdefault.jpg" $tb }}
+{{ $dt := $mx | base64Encode }}
```

Comment:
I'm not sure about what this is. This is an encoding of the URL? I cannot see how that can give us info about the image itself (size etc.)

Reply:
It gets the base64 value of [this](http://i.ytimg.com/vi/9MznJdyOl_E/maxresdefault.jpg) image that YT loads when there is no actual `maxresdefault.jpg`

The base64 value of this image is a given so if its base64 is present we will load the `hqdefault.jpg`

Reply:
```bash
{{ $dt := $mx | base64Encode }}
```

Are you saying that mx contains the image? And not the URL to the image? I may need a good nights sleep ...

Reply:
I have tested this locally and it fetches the correct thumb for the low-res videos.

Also I removed the CSS edits from this PR.

Reply:
@bep 

`$dt` stands for detect 
`$mx` stands for `maxresdefault.jpg` 

The base64 value of [this](http://i.ytimg.com/vi/9MznJdyOl_E/maxresdefault.jpg) image is `Ly9pLnl0aW1nLmNvbS92aS85TXpuSmR5T2xfRS9tYXhyZXNkZWZhdWx0LmpwZw==`

So if that value is present then we skip `maxresdefault.jpg` and load `hqdefault.jpg`

Reply:
>I have tested this locally and it fetches the correct thumb for the low-res videos.

But does it work the other way around?

Reply:
Why should it work the other way around?

YouTube loads this image when an actual maxresdefault.jpg is not available for low-resolution videos.

Reply:
Ok. I just figured out why I was getting the correct thumbnail locally.

The decoded base64 value is for the url `//i.ytimg.com/vi/9MznJdyOl_E/maxresdefault.jpg`

And it's for that specific video. Doh!

I'm closing this PR. Sorry to waste your time. I'll look into this a bit more with a clear head tomorrow.